pkg/util/oc/pod_commands.go

```diff
@@ -83,3 +90,7 @@ func WaitAllPodsReady(t test.TestHelper, ns string) {
 	t.T().Helper()
 	shell.Executef(t, `oc -n %s wait --for condition=Ready --all pods --timeout 180s`, ns)
 }
+func WaitCondition(t test.TestHelper, ns string, kind string, name string, condition string) {
+	t.T().Helper()
+	shell.Executef(t, `oc wait -n %s %s/%s --for condition=%s  --timeout %s`, ns, kind, name, condition, "10s")
+}
```

Comment:
When I call this function, I expect it to wait longer than 10s. The retry should be performed inside this function and it should log something every 10s.

Reply:
Done